config/config.go

```diff
@@ -91,10 +94,24 @@ func GetOptionalParameters(config *Config, defaultLidarDataRateMsec, defaultMapR
 		} else {
 			lidarDataFreqHz, err := strconv.Atoi(strCameraDataFreqHz)
 			if err != nil {
-				return 0, 0, newError("camera[data_frequency_hz] must only contain digits")
+				return 0, "", 0, 0, newError("camera[data_frequency_hz] must only contain digits")
 			}
 			lidarDataRateMsec = 1000 / lidarDataFreqHz
 		}
+		imuName, ok = config.MovementSensor["name"]
```

Comment:
instead of 'ok', let's say 'exists', since as a reader it's surprising to see an `if ok` without an `else` to handle the "not ok" case